terminal/terminal_other.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
You're right, sorry about that! Must've seen that pattern before and mistook its significance.